src/client.toit

```diff
@@ -148,16 +157,19 @@ class Client:
     before returning.
   */
   subscribe topic_filters/List:
+    if is_closed: throw CLIENT_CLOSED_EXCEPTION
     packet_id := next_packet_id_++
-
-    packet := SubscribePacket
-      topic_filters
-      --packet_id=packet_id
-
+    packet := SubscribePacket topic_filters --packet_id=packet_id
     wait_for_ack_ packet_id: | latch/monitor.Latch |
       send_ packet
       ack := latch.get
-      if not ack: throw "client closed"
+      if not ack: throw CLIENT_CLOSED_EXCEPTION
+
+  /**
+  Unsubscribe to a single topic $filter.
+  */
+  unsubscribe filter/string -> none:
+    // Not implemented yet.
```

Comment:
?
```suggestion
    // Not implemented yet.
    throw "UNIMPLEMENTED"
```

Reply:
I am already using this. Will implement soon.